src/App.jsx

```diff
@@ -4,22 +4,47 @@ import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
 
+const url = `https://api.airtable.com/v0/${import.meta.env.VITE_AIRTABLE_BASE_ID}/${import.meta.env.VITE_TABLE_NAME}`;
+
+const headers = {
+  Authorization: `Bearer ${import.meta.env.VITE_AIRTABLE_API_TOKEN}`,
+  'Content-Type': 'application/json',
+};
+
+
 const App = () => {
 
   const [todoList, setTodoList] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
 
+
+  const fetchData = async () => {
+    const options = {
+      method: "GET",
+      headers: headers,
+    };
+    try {
+      const response = await fetch(url, options);
```

Comment:
add a setIsLoading(false) here because at this point the data has finished loading

Reply:
Done